proto/ibc/core/channel/v1/channel.proto

```diff
@@ -114,9 +114,11 @@ message Packet {
   uint64 timeout_timestamp = 8 [(gogoproto.moretags) = "yaml:\"timeout_timestamp\""];
 }
 
-// PacketAckCommitment defines the genesis type necessary to retrieve and store
-// acknowlegements.
-message PacketAckCommitment {
+// PacketState defines the generic type necessary to retrieve and store
+// packet commitments, acknowledgments, and receipts
+// Caller is responsible for knowing the context necessary to interpret this
+// state as a commitment, acknowledgement, or a receipt
+message PacketState {
```

Comment:
`PacketAckCommitment` was a confusing name, changed it to `PacketState` with docs